apps/e2e/ag-grid-storybook/src/app/data-entry-grid/data-entry-grid.component.html

```diff
@@ -27,7 +27,7 @@ <h1>Data Entry Grid</h1>
 ></ng-container>
 
 <br />
-<sky-icon icon="check" iconType="fa"></sky-icon>
+<app-font-loading></app-font-loading>
```

Comment:
Instead of writing text/icons directly to the DOM, could we create a service that wraps `FontFaceObserver`? That way, the loading is done without DOM manipulation and could be tied into your existing `ready` observable. Using a service would also isolate the font-loading functionality to the Angular code, and Cypress wouldn't have to know about it. I'm curious about your thoughts since you have more familiarity with Cypress than I do. 

Something like: 
```
import { Injectable } from '@angular/core';

import FontFaceObserver from 'fontfaceobserver';

@Injectable({ providedIn: 'root' })
export class SkyE2eFontLoaderService {
  public async load(): Promise<void> {
    try {
      await Promise.all(
        [
          'BLKB Sans',
          'BLKB Sans Condensed',
          'Blackbaud Sans',
          'FontAwesome',
          'skyux-icons',
        ].map((f) => new FontFaceObserver(f))
      );
      console.log('Fonts loaded.');
    } catch (err) {
      console.error('Failed to load fonts.');
    }
  }
}

// In the ag-grid story component (rudimentary example):
await this.#fontLoaderSvc.load();
this.ready = true;

```

Reply:
Good call. Added this change.